video/external/src/backend.rs

```diff
@@ -85,29 +85,37 @@ impl ExternalVideoBackend {
 
         let (filename, md5sum) = Self::get_openh264_data()?;
 
-        let filepath = std::env::current_exe()?
+        let current_exe = std::env::current_exe()?;
+        let directory = current_exe
             .parent()
-            .ok_or("Could not determine Ruffle location.")?
-            .join(filename);
+            .ok_or("Could not determine Ruffle location.")?;
+        let filepath = directory.join(filename);
 
         // If the binary doesn't exist in the expected location, download it.
         if !filepath.is_file() {
             let url = format!("{}{}{}", URL_BASE, filename, URL_SUFFIX);
             let response = reqwest::blocking::get(url)?;
-            let bytes = response.bytes()?;
-            let mut bzip2_reader = BzDecoder::new(bytes.as_ref());
+            let mut bzip2_reader = BzDecoder::new(response);
 
-            let mut file = File::create(filepath.clone())?;
-            copy(&mut bzip2_reader, &mut file)?;
+            let mut tempfile = tempfile::NamedTempFile::with_prefix_in(filename, directory)?;
+            copy(&mut bzip2_reader, &mut tempfile)?;
+            // Let's assume that if this fails, it's because another process has already put it there
```

Comment:
I guess...?

Reply:
(Also see next line. Sounds good, @kjarosh?)

Reply:
Yup, sounds good. I'd also mention in the comment that the MD5 will be checked nonetheless